bike-erp/backend/src/main/services/authenticationService/AuthenticationService.ts

```diff
@@ -41,7 +41,7 @@ export class AuthenticationService {
         const account = await AuthenticationService.accountDao.fetchAccount(email);
 
         //Verifying if there was any account with the same email in the database
-        if (account.length == 0) {
+        if (account.length === 0) {
```

Comment:
Good change on this point, should make the code more strict

Reply:
I agree with Simon!